gradle/libs.versions.toml

```diff
@@ -1,11 +1,11 @@
 [versions]
 cacheApiVersion = "1.1.1"
 springDataVersion = "3.0.12"
-springFrameworkVersion = "6.0.16"
+springFrameworkVersion = "6.0.18"
 springShiroVersion = "1.12.0"
 springShellVersion = "1.2.0.RELEASE"
 springBootVersion = "3.0.13"
-springTestGemFireVersion = "1.0.3"
+springTestGemFireVersion = "1.0.4"
```

Comment:
I thought we had removed the dependency that SDG had on Spring Data Test GemFire?